includes/admin/class-wp-job-manager-settings.php

```diff
@@ -431,7 +431,17 @@ public function output() {
 				jQuery(this).addClass('nav-tab-active');
 				return false;
 			});
-			jQuery('.nav-tab-wrapper a:first').click();
+			var goto_hash = window.location.hash;
+			if ( goto_hash ) {
+				var the_tab = jQuery( 'a[href="' + goto_hash + '"]' );
+				if ( the_tab ) {
```

Comment:
This should be:
`if ( the_tab.length > 0 ) {`

jQuery's selector will always return an object that would evaluate `true` in the current line.